WD-Firmware/comrv/comrv_hooks.c

```diff
@@ -0,0 +1,139 @@
+/* 
+* SPDX-License-Identifier: Apache-2.0
+* Copyright 2019 Western Digital Corporation or its affiliates.
+* 
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+* 
+* http:*www.apache.org/licenses/LICENSE-2.0
+* 
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+/**
+* @file   comrv_hooks.c
+* @author Ronen Haen
+* @date   20.10.2019
+* @brief  The file implements the COM-RV hooks interfaces
+* 
+*/
+
+/**
+* include files
+*/
+#include "common_types.h"
+#include "string.h"
+#include "comrv_api.h"
+/**
+* definitions
+*/
+
+/**
+* macros
+*/
+
+/**
+* types
+*/
+
+/**
+* globals
+*/
+extern void* __OVERLAY_GROUPS_START__;
```

Comment:
change name to overlay storage start address

Reply:
done in ca8dbd0